hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1092,18 +1090,23 @@ int SaraNcpClient::selectSimCard(ModemState& state) {
             } else if (r == 1 && static_cast<UbloxSaraUmnoprof>(umnoprof) == UbloxSaraUmnoprof::STANDARD_EUROPE) {
                 auto respBand = parser_.sendCommand(UBLOX_UBANDMASK_TIMEOUT, "AT+UBANDMASK?");
                 uint64_t ubandCatm1 = 0;
-                auto retBand = CHECK_PARSER(respBand.scanf("+UBANDMASK: 0,%llu", &ubandCatm1));
+                char ubandCatm1_str[24] = {};
```

Comment:
Nit: `camelCase`.

Reply:
`ubandCatm1Str` is not as readable, so I would prefer `ubandCatm1_str`.  I could rename them to `ubandUint` and `ubandStr` I guess.